modular_skyrat/modules/contractor/code/items/boxes.dm

```diff
@@ -73,4 +73,49 @@
 	new /obj/item/extraction_pack/contractor(src)
 	new /obj/item/fulton_core(src)
 
+/obj/item/storage/box/syndicate/contract_kit/midround
+	name = "Contract Kit"
+	special_desc = "Supplied to Syndicate contractors."
+	special_desc_requirement = EXAMINE_CHECK_CONTRACTOR
+	icon_state = "syndiebox"
+	illustration = "writing_syndie"
+
+/obj/item/storage/box/syndicate/contract_kit/PopulateContents()
+	// All about 4 TC or less - some nukeops only items, but fit nicely to the theme.
```

Comment:
If there's some nukeops-only items, it's probably usually for good reasons, no? They're meant to be contractors, not nuke ops, they should ideally not often have to go loud.

Reply:
i mean, all of those items assist in stealth no? 
Thermal glasses allow you to see your target / avoid security more effectivelly. 
Implantable radio allows you to communicate with other traitors to pinpoint important information about your target or cooperate (and its a traitor item so i dont see a point in the highlight) 
For the sentience potion i have nothing to say.
Combat gloves+ allow you to silence your target and sweep your targets or securitys legs as you run away.
Also we already have elite syndicate suits, double energy swords and combat medkits avaible to progressive traitors... so i dont really see too much of an argument here to be made about some _random_ low-tier nuclear gear that helps non lethally

Reply:
> If there's some nukeops-only items, it's probably usually for good reasons, no? They're meant to be contractors, not nuke ops, they should ideally not often have to go loud.

First off, this was inherited TG code, so it was not fully my idea in the first place
Second of all, very few of the items are nukeops-only there (thanks to our skyrat changes). There is a whole _one_ nukeops-only item that's in the contractor random item list, being the syndicate sentience potion.